src/metabase/pulse/render/body.clj

```diff
@@ -243,19 +243,33 @@
    "#c589b9" "#efce8c" "#b5f95c" "#e35850" "#554dbf" "#bec589" "#8cefc6" "#5cc2f9" "#55e350" "#bf4d4f"
    "#89c3c5" "#be8cef" "#f95cd0" "#50e3ae" "#bf974d" "#899bc5" "#ef8cde" "#f95c67"])
 
+(defn- donut-info
+  "Process rows with a minimum slice threshold. Collapses any segments below the threshold given as a percentage (the
+  value 25 for 25%) into a single category as \"Other\". "
+  [threshold-percentage rows]
+  (let [total                    (reduce + 0 (map second rows))
+        threshold                (* total (/ threshold-percentage 100))
+        {as-is true clump false} (group-by (comp #(> % threshold) second) rows)
+        rows (cond-> as-is
+               (seq clump)
+               (conj [(tru "Other") (reduce + 0 (map second clump))]))]
+    {:rows        rows
+     :percentages (into {}
+                        (for [[label value] rows]
+                          [label (if (zero? total)
+                                   (tru "N/A")
+                                   (let [f (DecimalFormat. "###,###.##%")]
```

Comment:
You should look at the `custom-formatting` Setting's value of `percent` for the format string to use here... see the code @noahmoss wrote for the XLSX stuff for example.

We're not doing it anywhere else but falling back to 
```clj
(java.text.NumberFormat/getPercentInstance (metabase.util.i18n/user-locale))
```

might be better than assuming English-style formatting although that's really a separate question.

Reply:
Actually I think this could show up in viz settings as well (?). So it should be viz settings > `custom-formatting` Setting > default to get the format string

Reply:
We probably shouldn't do any number formatting in pulses based on user locale, right? Partly because we don't do it on the frontend, but moreso because a pulse could be sent to multiple users with different locales.

Reply:
yeah seems like the server sets it regardless of where the email goes. The format string gets localized though. `###,###.##%` will use the locale's separators and groupings.

> The characters listed here are used in non-localized patterns. Localized patterns use the corresponding characters taken from this formatter's DecimalFormatSymbols object instead, and these characters lose their special status. Two exceptions are the currency sign and quote, which are not localized. 
https://docs.oracle.com/javase/7/docs/api/java/text/DecimalFormat.html